### ISSUE TITLE
Provider INWX wrong error message

### ISSUE THREAD

**Issue by pomeloy** [qdm12/ddns-updater#494]
**TLDR**: error message using DNS provider INWX even though records were successfully updated

1. Is this urgent: No
2. DNS provider(s) you use: INWX 
3. Program version: 

    

    `2023-06-17T08:41:18.419Z (commit 828373d)`

4. What are you using to run the container: docker-compose
5. Extra information (optional)

Somehow this error message check fails. [https://github.com/qdm12/ddns-updater/blob/828373da7f95289fe6a9504a48d99bee6a723a94/internal/provider/providers/inwx/provider.go#L136-L137](https://github.com/qdm12/ddns-updater/blob/828373da7f95289fe6a9504a48d99bee6a723a94/internal/provider/providers/inwx/provider.go#L136-L137)

Logs:

```log
2023-06-17T12:00:52Z INFO IPv4 address of *** is *** and your IPv4 address is ***
2023-06-17T12:00:52Z INFO Updating record [domain: *** | host: *** | provider: INWX] to use ***
2023/06/17 12:00:54 "GET http://***:8000/ HTTP/1.1" from *** - *** in 157.679µs
2023-06-17T12:00:54Z ERROR unknown response received: good
```

Configuration file (**remove your credentials!**):

```json
{
    "settings": [
        {
            "provider": "inwx",
            "domain": "",
            "host": "",
            "username": "",
            "password": "",
            "ip_version": "ipv4"
        }
    ]
}
```

Host OS: docker image: qmcgaw/ddns-updater

**Comment by qdm12** [comment]
Hopefully 7f43415d4eec1377dbb48b3468dc1d36c60ea025 fixes it in the latest image, please comment back here if it's not 😉

**Comment by qdm12** [comment]
Never mind, I overridden that commit with 27456b628d2f29b949d94013f6669a6dc4b4d54b

Basically the OR `||` should had been an AND `&&` in that if condition.
